cmd/karavictl/cmd/generate_admin_token.go

```diff
@@ -53,18 +48,12 @@ func NewAdminTokenCmd() *cobra.Command {
 				reportErrorAndExit(JSONOutput, cmd.ErrOrStderr(), err)
 				return err
 			}
-			if refExpTime <= 0 {
```

Comment:
Is this an unreachable condition?

Reply:
Moved the functionality in middleware layer.